Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Use NVIDIA CUDA as base image
+FROM nvidia/cuda:12.4.1-devel-ubuntu22.04
+
+# Set working directory
+WORKDIR /app
+
+# Set environment variables to non-interactive (this prevents some prompts)
+ENV DEBIAN_FRONTEND=non-interactive
+
+# Install required libraries, tools, and Python3
+RUN apt-get update && apt-get install -y libgl1 libglib2.0-0 ffmpeg curl git python3.10 python3-pip nvidia-cuda-toolkit
```

Comment:
Do we really need to install `nvidia-cuda-toolkit` if we use the CUDA docker image as the base? Can you check the list if everything is really needed? I think the dependency on `libgl1` was removed because we are using headless opencv now.

Reply:
It was needed for another image base. I removed it here.

Reply:
I removed both `libgl1`, `libglib2.0-0`